platform/linux-generic/odp_ishm.c

```diff
@@ -698,9 +698,11 @@ static int create_file(int block_index, huge_flag_t huge, uint64_t len,
 		}
 
 		if (ret == -1) {
-			_ODP_ERR("%s memory allocation failed: fd=%d, file=%s, "
-				"err=\"%s\"\n", (huge == HUGE) ? "Huge page" :
-				"Normal page", fd, filename, strerror(errno));
+			_ODP_LOG(use_huge ? ODP_LOG_WARN : ODP_LOG_ERR,
+				 use_huge ? "WARN" : "ERR",
+				 "%s memory allocation failed: fd=%d, file=%s, err=\"%s\"\n",
+				 use_huge ? "Huge page" : "Normal page", fd, filename,
```

Comment:
"page" could be moved to the common part.

Reply:
Changed in V2.